composer.json

```diff
@@ -24,6 +24,7 @@
         "php": ">=5.4.8",
         "symfony/event-dispatcher": "~2.3",
         "symfony/options-resolver": "~2.3",
+        "symfony/filesystem": "~2.3",
```

Comment:
Should we depend on this or reimplement the code locally? The filesystem component provides a more resilient `symlink` method and recursive delete.